test/java/org/apache/ivy/plugins/parser/xml/XmlModuleUpdaterTest.java

```diff
@@ -44,13 +45,21 @@
 import org.apache.ivy.plugins.repository.BasicResource;
 import org.apache.ivy.util.FileUtil;
 import org.xml.sax.SAXParseException;
+import org.apache.ivy.util.XMLHelper;
 
 public class XmlModuleUpdaterTest extends TestCase {
-    
+    Properties p = (Properties) System.getProperties().clone();
+
+    protected void setUp() {
+        System.setProperty(XMLHelper.ALLOW_DOCTYPE_PROCESSING, "true");
+        System.setProperty(XMLHelper.EXTERNAL_RESOURCES, "all");
```

Comment:
@bodewig It seems we need to allow doctype processing on all resources to run those tests.

Reply:
Interesting, "works on my machine" again. Which version and vendor of the JDK are you using?

Reply:
Ah, I now see both tests failing on OpenJDK 8, they do pass with OpenJDK 11. Will try to understand why before I enable "all" here.

Reply:
see https://issues.apache.org/jira/browse/IVY-1648 fixed with https://github.com/apache/ant-ivy/commit/503aafc594b777ffb035a457ed341e31334c23a4

Please let me know if you want me to credit you in a different way than https://github.com/apache/ant-ivy/commit/503aafc594b777ffb035a457ed341e31334c23a4#diff-aa24958121675fdba6b00fe830e2975de732eddfc94398f248ff8ac31f8bfb0cR46 @adpi2